[2025-05-06T21:00:02.980+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-06T21:00:03.011+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-06T20:00:00+00:00 [queued]>
[2025-05-06T21:00:03.020+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-06T20:00:00+00:00 [queued]>
[2025-05-06T21:00:03.022+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-06T21:00:03.035+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_dbt> on 2025-05-06 20:00:00+00:00
[2025-05-06T21:00:03.052+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2403) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-06T21:00:03.052+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_etl_dag', 'run_dbt', 'scheduled__2025-05-06T20:00:00+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmp9c3m7lz_']
[2025-05-06T21:00:03.054+0000] {standard_task_runner.py:63} INFO - Started process 2405 to run task
[2025-05-06T21:00:03.055+0000] {standard_task_runner.py:91} INFO - Job 115: Subtask run_dbt
[2025-05-06T21:00:03.073+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-06T21:00:03.109+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl_dag.run_dbt scheduled__2025-05-06T20:00:00+00:00 [running]> on host c6817411f6c2
[2025-05-06T21:00:03.208+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='run_dbt' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-06T20:00:00+00:00'
[2025-05-06T21:00:03.209+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-06T21:00:09.420+0000] {logging_mixin.py:188} INFO - dbt run failed: Command '['dbt', 'run']' returned non-zero exit status 1.
[2025-05-06T21:00:09.421+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-06T21:00:09.423+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-06T21:00:09.436+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=run_dbt, run_id=scheduled__2025-05-06T20:00:00+00:00, execution_date=20250506T200000, start_date=20250506T210003, end_date=20250506T210009
[2025-05-06T21:00:09.531+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-06T21:00:09.565+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-06T21:00:09.602+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
